docs/architecture/Security_Assessment.md

```diff
@@ -60,12 +60,32 @@ N/A
 
 ## Threats & Risks
 
-tbd
+N/A - No direct threats & vulnerabilities detected during the assessment, taking into account already implemented security controls & requirements.
 
 ### Mitigated Threats
 
 N/A
 
+### Implemented Security Controls
+
+    Authentication & Authorization Concept
+        As per the Association "business requirements", SSI Authority & Schema Registry Product is publicly available, even if the customers are not part of Catena-X. Therefore no authentication, authorization & session management concepts were implemented.
+        
+    Data Storage & Encryption
+        Data Stored within the Registry DB (Postgres) is publicly available and not confidential information. No encryption requirements for data at rest is required.
+        
+    API Security
+        API is publicly available and may be accessed by Everyone.
+        Two endpoints are available, for both functionalities of the application :
+            1. Read-only Get Endpoint - to receive information about available credentials and their authorities
+            2. Read-only Post Endpoint - to validate credential schemas
+        Rate limitting configuration is available, once properly configured by the "Operating Company" will grant the availability controls for the application.
+    
+    Logging & Monitoring
+        All actions & requests performed by the Customers are logged in and stored within the application database with possibility for further audit, investigation & active monitoring, which may be configured by the "Operating Company"
+
```

Comment:
could you please provide this in bullet point structure, currently it looks like this:
![image](https://github.com/user-attachments/assets/95e48fd6-0244-495b-8975-6db87057009b)
and please also remove the two unneeded empty lines.

Reply:
I have updated the text.

@szymonkowalczykzf fyi